packages/suite/src/views/wallet/receive/index.tsx

```diff
@@ -6,6 +6,23 @@ import FreshAddress from './components/FreshAddress';
 import UsedAddresses from './components/UsedAddresses';
 import { isPending } from '@wallet-utils/transactionUtils';
 import { getAccountTransactions } from '@wallet-utils/accountUtils';
+import { variables } from '@trezor/components';
+import styled from 'styled-components';
```

Comment:
Do not change now.

I prefer to have:
_imports of libraries_ (import styled from 'styled-components';)
_space_
_imports of project components, utils,.._ (import { isPending } from '@wallet-utils/transactionUtils';)
_space_
_imports of types_ (import type { FeedbackType } from '@suite-types/guide';)

Reply:
I personally don't see the benefit of bothering with the order of imports, especially if it's not consistent across the whole project (and automatized...).

Reply:
I did not want to bother with it. Maybe we could try to automate it because I think that it makes imports clearer.